client/joinus_client/src/components/App.js

```diff
@@ -5,7 +5,7 @@ import Navbar from "./Navbar";
 import Home from "./Home";
 import FunPhoto from "./Home/FunPhotos";
 import Chat from "./Chat";
-import Userpage from "./UserPage/index";
```

Comment:
rename the file to index.js instead of Index.js

Reply:
index.js broke my build, where as Index.js didn't 🤷

I get this error with index.js :

```
Compiled with problems:X

ERROR in ./src/components/App.js 13:0-40

Module not found: Error: Cannot find file: 'index.js' does not match the corresponding name on disk: './src/components/UserPage/Index.js'.
```

Reply:
change them both to lower case

Reply:
oml, classic talking to the duck - Fixed now